packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain/SourceNetworkBox.tsx

```diff
@@ -57,12 +52,25 @@ function Amount2ToggleButton({
   )
 }
 
+export const useAmount2InputVisibility = create<{
+  isAmount2InputVisible: boolean
+  showAmount2Input: () => void
+}>(set => ({
+  isAmount2InputVisible: false,
+  showAmount2Input: () => {
+    set(() => ({
+      isAmount2InputVisible: true
+    }))
+  }
+}))
```

Comment:
Any chance we can rename amount2 to be more descriptive? And if we start reusing it in multiple place, I would probably extract it

Reply:
We use amount2 everywhere for batch transfers